angular.json

```diff
@@ -105,7 +105,7 @@
         "extract-i18n": {
           "builder": "@angular-devkit/build-angular:extract-i18n",
           "options": {
-            "browserTarget": "core:build"
+            "buildTarget": "core:build"
```

Comment:
what is the difference between those targets?

Reply:
else lgtm

Reply:
dunno, I ran `ng update` on the workspace and it did this

Reply:
Well then, LGTM